index.js

```diff
@@ -1,159 +1,98 @@
-"use strict"
+'use strict';
 
 module.exports = {
-    "extends": "stylelint-config-standard",
-    "customSyntax": "scss",
-    "plugins": [
-        "stylelint-scss"
-    ],
-    "ignoreFiles": [
-        "sass/vendor/*.scss"
-    ],
-    "rules": {
-        "at-rule-empty-line-before": [
-            "always",
+    'extends': 'stylelint-config-standard-scss',
+    'customSyntax': 'postcss-scss',
+    'rules': {
+        'alpha-value-notation': 'number',
+        'at-rule-empty-line-before': [
+            'always',
             {
-                "except": [
-                    "blockless-after-same-name-blockless",
-                    "first-nested"
-                ],
-                "ignore": [
-                    "after-comment"
-                ],
-                "ignoreAtRules": [
-                    "extend",
-                    "include",
-                    "else",
-                    "elseif",
-                    "content"
+                except: ['blockless-after-blockless', 'first-nested'],
+                ignore: ['after-comment'],
+                ignoreAtRules: [
```

Comment:
I added them as it was more annoying to have for this scss `@` rules spacing. If you want to readd them I'm fine to overwrite the rule in an own set.

Reply:
Example:

here I know that this was not possible with this rule and the rule did just force unnecessary empty lines before this scss `@`.

```js
    @if $test == 1 {
        color: red;
    } @else if $test == 2 {
        color: blue;
    } @else {
        color: purple;
    }
```

Some I thing as here for the `@content` scss:

```scss
@mixin some($name) {
  @media {
    @content;
  }
}
```

So that is why I recommend not adding `@` enforcing to scss `@` methods. But maybe stylelint changed here to know what is `scss` and what is `css`.